tests/Unit/Evolution/DgSubcell/Test_Mesh.cpp

```diff
@@ -3,13 +3,34 @@
 
 #include "Framework/TestingFramework.hpp"
 
+#include <sstream>
+#include <string>
+
+#include "DataStructures/DataVector.hpp"
 #include "Evolution/DgSubcell/Mesh.hpp"
 #include "NumericalAlgorithms/Spectral/Basis.hpp"
 #include "NumericalAlgorithms/Spectral/Mesh.hpp"
 #include "NumericalAlgorithms/Spectral/Quadrature.hpp"
 #include "NumericalAlgorithms/Spectral/Spectral.hpp"
 
 namespace {
+void print_comparison_point_computation() {
+  // Prints the grid spacing an FD points for different options of choosing
+  // the "right" number of FD points for the number of DG points.
```

Comment:
I don't understand how the comment is related to the code; where are the different options?

Reply:
I completely rewrote this to try and make it intelligible